frontend/src/components/PrevPointInput.js

```diff
@@ -0,0 +1,32 @@
+// TODO:
+/**
+ * 1. Breakup into participant and visit child components
+ * 2. Break fields into smaller individual components for re-use in other forms
+ * 3. Break up state in MobX for field re-use
+ */
+
+/* eslint-disable indent */
+import React from "react"
+import PropTypes from "prop-types"
+import Input from "@material-ui/core/Input"
+
+const PrevPointInput = props => {
+  return (
+    <Input
+      id={props.id}
+      name={props.name}
+      value={props.val}
+      onChange={e => props.changeFunc(e.target.value)}
```

Comment:
Ill approve as is but I see down the line components being interested in more than just the value from the action.

Reply:
Should allow for just the event to be emitted now. Check out what I pushed and see if it lines up,